files/build_templates/docker_image_ctl.j2

```diff
@@ -523,6 +531,8 @@ start() {
         {%- if docker_container_name == "database" %}
         NET="bridge"
         DB_OPT=$DB_OPT" -v /var/run/redis$DEV:/var/run/redis:rw "
+	{%- elif docker_container_name == "snmp" %}
+        NET="bridge"
```

Comment:
>NET="bridge" [](http://example.com/codeflow?start=8&length=12)

Checking best practice ([https://docs.docker.com/network/network-tutorial-standalone/](https://docs.docker.com/network/network-tutorial-standalone/)), default bridge network is not best choice for production. User-defined bridge networks is best choice for production. Can we use user-defined bridge network here?

Reply:
I agree, defining a user-defined bridge is the best case solution for this item.
Problem is - this FR wasn't scoped to define this user-bridge and therefor remained in the general bridge domain.
If such a bridge will be defined - then we can add it to the SNMP container and redis container (and any future container) as well.